gbpservice/nfp/service_vendor_agents/vyos/vyos_session/utils.py

```diff
@@ -68,6 +69,13 @@ def init_logger(logger):
     # add the handlers to the logger
     logger.addHandler(file_handler)
 
+    formatter = logging.Formatter('vyos %(name)s %(funcName)s() %(levelname)s '
+                                  '%(message)s')
+    sys_handler = handlers.SysLogHandler(address=('localhost', 514))
+    sys_handler.setFormatter(formatter)
+    sys_handler.setLevel(logging.DEBUG)
+    logger.addHandler(sys_handler)
```

Comment:
There is an existing file handler and we added this sys_handler. Are the logs now landing in two different files or single one?